tests/integration/targets/mso_schema_site_anp_epg_staticport/tasks/main.yml

```diff
@@ -512,6 +513,85 @@
     that:
     - nm_remove_again_stat2e2 is not changed
 
+# ADD STATIC PORT WITHOUT VLAN
+- name: Add static port 3 (dpc) to EPG6 of AP2 without VLAN (error)
+  cisco.mso.mso_schema_site_anp_epg_staticport:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: '{{ mso_site | default("ansible_test") }}'
+    template: Template 1
+    anp: AP2
+    epg: EPG6
+    pod: pod-2
+    leaf: 102
+    path: eth1/3
+    deployment_immediacy: lazy
+    mode: regular
+    type: dpc
+    primary_micro_segment_vlan: 199
+    state: present
+  ignore_errors: true
+  register: nm_add_stat3e6
+
+- name: Verify nm_add_stat3e6
+  ansible.builtin.assert:
+    that:
+    - nm_add_stat3e6 is failed
+    - nm_add_stat3e6.msg == "state is present or absent but all of the following are missing{{":"}} vlan."
+
+# ADD STATIC PORT WITH VLAN
+- name: Add static port 3 (dpc) to EPG6 of AP2 with VLAN (success)
+  cisco.mso.mso_schema_site_anp_epg_staticport:
+    <<: *mso_info
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    site: '{{ mso_site | default("ansible_test") }}'
+    template: Template 1
+    anp: AP2
+    epg: EPG6
+    pod: pod-2
+    leaf: 102
+    path: eth1/3
+    vlan: 100
+    deployment_immediacy: lazy
+    mode: regular
+    type: dpc
+    primary_micro_segment_vlan: 199
+    state: present
+  register: nm_add_stat3e6
+
+- name: Verify nm_add_stat3e6
```

Comment:
assert for add seems redundant as we already test creation with its error and this has not changed in your contribution